project-base/app/tests/FrontendApiBundle/Functional/Brand/BrandSearchTest.php

```diff
@@ -4,16 +4,19 @@
 
 namespace Tests\FrontendApiBundle\Functional\Brand;
 
+use Ramsey\Uuid\Uuid;
 use Shopsys\FrameworkBundle\Component\Translation\Translator;
 use Tests\FrontendApiBundle\Test\GraphQlTestCase;
 
 class BrandSearchTest extends GraphQlTestCase
 {
     public function testBrandSearch(): void
     {
+        $userIdentifier = Uuid::uuid4()->toString();
+
         $query = '
             query {
-                brandSearch(search: "de") {
+                brandSearch(searchInput: { search: "de", userIdentifier: "' . $userIdentifier . '"}) {
```

Comment:
I am thinking, do we have to force users of the API to provide `userIdentifier` (and `isAutocomplete`) input values everytime, even if they wont use LuigisBox? Would not it make sense to set some default values in SearchInputDecorator?

Reply:
Sadly there is no other way as I cannot decide in validation where this data will end and I cannot add validation error to Luigi's Box ProductsSearchResultsProvider as there is some mystery error that does not pass Exceptions to the output.